TACC/Core-Portal
# bug/FP-1154 -  Copy Modal default destination storage system fix

Removed code to set Copy modal in line with Move modal, it also was causing a bug if Community Data was unavailable.

## Overview: ##
The Copy modal onOpening was setting the default destination storage system to a system that didn't exist. Now, it the destination system is set to the same as the file's original location.  This is already done in the Move modal, so it brings the two modals into parity.

## Related Jira tickets: ##

* [FP-1154](https://jira.tacc.utexas.edu/browse/FP-1154)

## Summary of Changes: ##
Removed  systemParams from onOpened function, and pass file's existing tapis params.

## Testing Steps: ##
1. Open Copy modal for any file
2. The destination should be the same as the file's current location.
3. Copy should proceed successfully.

## UI Photos:

## Notes: ##
This primarily affects A2CPS.
